source/PreferencesPanel.cpp

```diff
@@ -72,6 +72,7 @@ namespace {
 	const string TARGET_ASTEROIDS_BASED_ON = "Target asteroid based on";
 	const string BACKGROUND_PARALLAX = "Parallax background";
 	const string ALERT_INDICATOR = "Alert indicator";
+	const string CONTROL_SHIP_WITH_MOUSE = "Control ship with mouse";
```

Comment:
This is for stuff that needs to be handled differently to normal, usually because those strings are going to need to be referred to in multiple places.
This string is only ever used once, just put the literal there, in the SETTINGS array.